mscore/pluginCreator.cpp

```diff
@@ -430,15 +436,16 @@ void PluginCreator::load()
 //   savePlugin
 //---------------------------------------------------------
 
-void PluginCreator::savePlugin()
+void PluginCreator::doSavePlugin(bool saveas) 
       {
-      if (created) {
+      if (saveas) {
             path = mscore->getPluginFilename(false);
             if (path.isEmpty())
                   return;
-            }
+      }
```

Comment:
indent...

Reply:
you forgot tzhis one (but rightfully fixed another and older one a few lines down)